agent/src/NullBuffer.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace Laravel\NightwatchAgent;
+
+class NullBuffer
```

Comment:
What do you think about creating the `Buffer` contract/interface to ensure `StreamBuffer` and `NullBuffer` implement the same interface?

Reply:
I don't like to create interfaces unless we anticipate 3rd party implementations or testing fakes.

Because we own both of these I don't think it adds any value. Out test suite and static analysis will tell us if there is an API issue between the two.